.github/workflows/ci.yaml

```diff
@@ -125,6 +124,13 @@ jobs:
           then
             # Set opensearch kernel parameters
             sudo sysctl -w vm.max_map_count=262144 vm.swappiness=0 net.ipv4.tcp_retries2=5
+            printf '\nDisk usage before cleanup\n'
+            df --human-readable
+            # Based on https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+            rm -r /usr/share/dotnet
+            rm -r /opt/hostedtoolcache/
+            printf '\nDisk usage after cleanup\n'
+            df --human-readable
```

Comment:
Looks like the issue with OS blocking was related to running low on disk space. Going forward we should most probably switch to the reusable workflow.

Reply:
I guess an issue should do.